plugins/email_service/app/models/email_service/plain_email.rb

```diff
@@ -0,0 +1,105 @@
+# module EmailService
```

Comment:
What is the point of keeping all this commented files?

Reply:
Removed in the subsequent commit.